lib/phoenix_live_component.ex

```diff
@@ -186,7 +184,7 @@ defmodule Phoenix.LiveComponent do
   for each card, passing the card struct as argument to CardComponent:
 
       <%= for card <- @cards do %>
-        <%= live_component @socket, CardComponent, card: card, board_id: @id %>
+        <%= live_component @socket, CardComponent, card: card, id: card.id, board_id: @id %>
```

Comment:
without `:id`, events are not handled by LiveComponent as mentioned in the documentation.